src/components/global-header/global-header-test.stories.tsx

```diff
@@ -54,3 +54,44 @@ MenuWithIconOnlyButtonsStory.parameters = {
     },
   },
 };
+
+export const FullMenuExample = () => (
```

Comment:
Do we intend to have this example present in storybook as well? As currently storybook will recognise and render this as a test story.

If this isn't intentional, I'll suggest changing the default export on line 10, so we are only using the icon-button example for storybook:

```tsx
export default {
  title: "Global Header/Test",
  includeStories: ["MenuWithIconOnlyButtonsStory"],
  parameters: {
    info: { disable: true },
    chromatic: {
      disableSnapshot: true,
    },
    docs: {
      inlineStories: false,
      iframeHeight: 200,
    },
  },
} as ComponentMeta<typeof GlobalHeader>;
```